lottie-swift/src/Public/Animation/AnimationPublic.swift

```diff
@@ -0,0 +1,199 @@
+//
+//  AnimationPublic.swift
+//  lottie-swift
+//
+//  Created by Brandon Withrow on 2/5/19.
+//
+
+import Foundation
+import CoreGraphics
+
+public extension Animation {
+  
+  // MARK: Animation (Loading)
+  
+  /**
+   Loads an animation model from a bundle by it's name. Returns `nil` if an animation is not found.
+   
+   - Parameter name: The name of the json file without the json extension. EG "StarAnimation"
+   - Parameter bundle: The bundle in which the animation is located. Defaults to `Bundle.main`
+   - Parameter subdirectory: A subdirectory in the bundle in which the animation is located. Optional.
+   - Parameter animationCache: A cache for holding loaded animations. Optional.
+   
+   - Returns: Deseralized `Animation`. Optional.
+   */
+  public static func named(_ name: String,
```

Comment:
Xcode 10.2 will warn for all users: _'public' modifier is redundant for static method declared in a public extension_.